SCIM License Mapping (#7474)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

    <Lightbox src="/img/docs/dbt-cloud/access-control/mark-as-required.png" width="60%" title="Mark the fields as required." />    

You can now begin assigning users to your SCIM app in Entra ID!

## Manage user licenses with SCIM

> You can manage user license assignments with SCIM groups in dbt. This is accomplished using an attribute in your IdP environment, rather than settings in dbt, to ensure accurate license assignment.

Reviewer: > You can manage user license assignments with SCIM groups in dbt.

I find this sentence a little confusing - is this referring to our "normal" way of license mapping?

> You can manage user license assignments with SCIM groups in dbt. This is accomplished using an attribute in your IdP environment, rather than settings in dbt, to ensure accurate license assignment.

Author: ```suggestion
You can manage user license assignments via SCIM with a user attribute in your IdP environment. This ensures accurate license assignment as users are provisioned in the IdP and onboarded into your dbt account.
```

> You can manage user license assignments via SCIM with a user attribute in your IdP environment. This ensures accurate license assignment as users are provisioned in the IdP and onboarded into your dbt account.

Author: @nm0s Hopefully this is clearer!

Reviewer: That sounds great!

To use license management via SCIM, enable the feature under the **SCIM** section in the **Single sign-on** settings. This setting will enforce license type for a user based on their SCIM attribute and disable the license mapping and manual configuration set up in dbt.  
    <Lightbox src="/img/docs/dbt-cloud/access-control/scim-managed-licenses.png" width="60%" title="Enable SCIM managed user license distribution." />    

> ### Add license type attribute for Okta 

Reviewer: @aaronbini mind verifying these steps for Okta?

Reviewer 2: 👍 @nm0s @matthewshaver this looks spot on to me!

Author: Thanks all!

    <Lightbox src="/img/docs/dbt-cloud/access-control/mark-as-required.png" width="60%" title="Mark the fields as required." />    

You can now begin assigning users to your SCIM app in Entra ID!

## Manage user licenses with SCIM

You can manage user license assignments via SCIM with a user attribute in your IdP environment. This ensures accurate license assignment as users are provisioned in the IdP and onboarded into your dbt account.

To use license management via SCIM, enable the feature under the **SCIM** section in the **Single sign-on** settings. This setting will enforce license type for a user based on their SCIM attribute and disable the license mapping and manual configuration set up in dbt.  
    <Lightbox src="/img/docs/dbt-cloud/access-control/scim-managed-licenses.png" width="60%" title="Enable SCIM managed user license distribution." />

_We recommend that you complete the setup instructions for your identity provider prior to enabling this toggle in your dbt account. Once enabled, any existing license mappings in <Constant name="dbt_platform" /> will be ignored._

The recommended steps for migrating to SCIM license mapping are as follows:

> 1. Set up SCIM but keep toggle disabled so existing license mappings continue to work as expected.

Author: ```suggestion
1. Set up SCIM but keep the toggle disabled so existing license mappings continue to work as expected.
```

> 2. Configure license attributes in your IdP.

Author: ```suggestion
2. Configure license attributes in your Identity Provider (IdP).
```
